### PR TITLE
Bugfix/requirements location

### PR DESCRIPTION
**Pull request by DorAbu**
🎓 Added more information to the README.md file
⚒ Fixed requirements location in pyproject.toml

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -26,7 +26,10 @@ classifiers = [
 ]
 description-file = "README.md"
 requires-python = ">=3.6"
-
```
Comment (DavidMeu) [inline pyproject.toml:29]: Maybe you should add here:
`[build-system]`
[build-system-table](https://www.python.org/dev/peps/pep-0518/#build-system-table)

**Review by DavidMeu — CHANGES_REQUESTED** [review]
[One comment.](https://github.com/Cyberproof/codemate/pull/3/commits/cd58427e3b9490e9c5b05087b35639b30fbb4b28#r666181570)

File: pyproject.toml (hunk 1 above)
Comment (DorAbu) [reply pyproject.toml:29]: We already have it at the beginning of the file
image

Comment (DavidMeu) [reply pyproject.toml:29]: Consider using poetry:
`[tool.poetry.dev-dependencies]`
[example](https://python-poetry.org/docs/dependency-specification/)
[pyproject.toml with poetry](https://python-poetry.org/docs/pyproject/)
[poetry](https://python-poetry.org/)

**Review by DavidMeu — APPROVED** [review]
Consider using:
[pyproject.toml with poetry](https://python-poetry.org/docs/pyproject/)
[poetry](https://python-poetry.org/)

**Pull request merged**